scripts/zones/Oldton_Movalpolos/mobs/Goblin_Wolfman.lua

```diff
@@ -0,0 +1,21 @@
+-----------------------------------
+-- Area: Oldton Movalpolos
+--  MOB: Goblin Wolfman
+-----------------------------------
+require("scripts/zones/Oldton_Movalpolos/MobIDs");
+require("scripts/globals/settings");
+require("scripts/globals/status");
+-----------------------------------
+
+function onMobInitialize(mob)
+    mob:setMobMod(dsp.mobMod.IDLE_DESPAWN, 55);
```

Comment:
make sure the value for that mobmod is retail accurate. I don't think 55 seconds is correct. its usually 3 or 5 minutes